src/main/java/com/google/api/generator/gapic/composer/GrpcServiceCallableFactoryClassComposer.java

```diff
@@ -72,6 +72,7 @@ public GapicClass generate(Service service, Map<String, Message> ignore) {
 
     ClassDefinition classDef =
         ClassDefinition.builder()
+            .setFileHeader(FileHeader.APACHE_LICENSE)
```

Comment:
Instead of changing this in each class, let's change all the ClassDefinitions at once in Composer.

Reply:
Not sure we can move the header addition to Composer, because in Composer class, we only `generateGenericClass` there, for class definition like `GrpcServiceCallableFactoryClassComposer`, it's [defined](https://github.com/googleapis/gapic-generator-java/blob/master/src/main/java/com/google/api/generator/gapic/composer/GrpcServiceCallableFactoryClassComposer.java#L73) in its own class.